### ISSUE TITLE
Missing Transmission icons

### ISSUE THREAD

**Issue by daltones** [keeferrourke/la-capitaine-icon-theme#111]
First of all I wanna thanks @keeferrourke for this great icon theme! 😃

Now the issue: I've noticed two icons missing from Transmission's GTK Frontend.

1. Button Options on the very bottom left (completely invisible):

La Capitaine:
![captura de tela de 2017-02-20 11-42-44](https://cloud.githubusercontent.com/assets/5603632/23130017/a2ead590-f763-11e6-984a-738529080ecf.png)

GNOME default icon theme:
![captura de tela de 2017-02-20 11-43-21](https://cloud.githubusercontent.com/assets/5603632/23130020/a79205fa-f763-11e6-8c94-cdb0d5846b5e.png)

2. “Closing Connections” icon on quitting Transmission:

This is La Capitaine. I think that's just a missing icon, falling back to GNOME default:
![captura de tela de 2017-02-20 11-44-06](https://cloud.githubusercontent.com/assets/5603632/23130105/f8e435b8-f763-11e6-9111-dd23d7825d60.png)

OS: Arch Linux
Transmission GTK: v2.92
La Captaine: `la-capitaine-icon-theme-git` from AUR, upgraded today.

**Comment by keeferrourke** [comment]
If you can figure what those icons are called, I'd be happy to add them as soon as I can.

**Comment by daltones** [comment]
I actually don't know the right way of discovering those icon names.

But looking at `gtk3-icon-browser` and `/usr/share/icons/Adwaita`, I guess the “closing connections” is called `network-idle`. (btw with `gtk3-icon-browser` we can see other missing icons, like more `network-(error|offline|receive|...)` icons)

The “options” icon I didn't found in `Adwaita`. Then I tried `numix-circle` and if I'm not wrong it doesn't have a icon for that too! But it falls back to default icon, don't getting invisible.

Thanks!

**Comment by keeferrourke** [comment]
Just took another look at `gtk3-icon-browser` and it seems that a lot of icons are unthemed, but I don't see any that are *missing* per se; that is, I don't see anything that fails to render or shows as a blank icon. Does this match the behaviour you're seeing?

**Comment by daltones** [comment]
@keeferrourke Yeah, all those network icons just don't have La Capitaine versions yet. This seems normal.

But that blank icon in button “options” I can't see an explanation. As I said, it's weird that even Numix Circle is unthemed in this case, but not blank.

**Comment by keeferrourke** [comment]
Are you by chance using the latest version from the master branch? There was a similar problem in KDE based environments re issues #106 and #107, but all the affected icons should have been fixed.

Perhaps pull the latest commits and let me know if there is still a problem?

**Comment by daltones** [comment]
Yeah, just tried.

Unfortunately it's not fixed yet. 😿

**Comment by demoy** [comment]
![screenshot_20170224_203406](https://cloud.githubusercontent.com/assets/2396524/23326824/9eac1b68-fad0-11e6-8a77-910aa3e288ef.png)


Interesting that the qt front end has shows the expected gnome ico. Maybe its another case of an outdated icon-cache.
